<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WidgetSendCommon</class>
 <widget class="QWidget" name="WidgetSendCommon">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>553</width>
    <height>170</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>6</number>
   </property>
   <property name="leftMargin">
    <number>9</number>
   </property>
   <property name="topMargin">
    <number>4</number>
   </property>
   <property name="rightMargin">
    <number>9</number>
   </property>
   <property name="bottomMargin">
    <number>9</number>
   </property>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>发送</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_6" stretch="5,1">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="24,1,1">
        <property name="spacing">
         <number>6</number>
        </property>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <property name="spacing">
           <number>0</number>
          </property>
          <item>
           <widget class="QTextEdit" name="textEdit"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <property name="spacing">
           <number>0</number>
          </property>
          <item>
           <widget class="QPushButton" name="btnSend">
            <property name="text">
             <string>发送</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnClearSend">
            <property name="text">
             <string>清除发送</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <property name="spacing">
           <number>6</number>
          </property>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_5">
            <item>
             <widget class="QCheckBox" name="isHexSend">
              <property name="text">
               <string>16进制发送</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="isEnterSend">
              <property name="text">
               <string>换行发送</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="isTimingSend">
              <property name="text">
               <string>定时发送</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <property name="spacing">
             <number>6</number>
            </property>
            <item>
             <widget class="QLabel" name="lblCycle">
              <property name="text">
               <string>周期</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="letCyCleTime"/>
            </item>
            <item>
             <widget class="QLabel" name="lblMS">
              <property name="text">
               <string>ms</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QLineEdit" name="letSendFilePath"/>
        </item>
        <item>
         <widget class="QPushButton" name="btnSelectSendFile">
          <property name="text">
           <string>选择文件</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnSendFile">
          <property name="text">
           <string>发送文件</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnClearSendFilePath">
          <property name="text">
           <string>清除发送</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
